modules/javafx.graphics/src/main/java/javafx/geometry/Point3D.java

```diff
@@ -410,10 +410,14 @@ public Point3D interpolate(Point3D endValue, double t) {
     }
 
     /**
-     * Returns a hash code value for the point.
-     * @return a hash code value for the point.
+     * Determines whether or not two objects are equal. Two instances of {@code Point3D}
```

Comment:
Since this object is known, I would start with "Determines if this point is equals to a given object".

Reply:
Agree. I propose the very similar wording "Determines whether this Point3D is equal to a given object."